src/shared/components/driverAnnotations/SettingsMenuButton.tsx

```diff
@@ -105,28 +106,63 @@ export default class SettingsMenuButton extends React.Component<
     }
 
     render() {
-        return (
-            <DefaultTooltip
-                trigger={[this.trigger]}
-                placement="bottomRight"
-                overlay={this.overlay}
-                visible={this.visibilityState}
-                onVisibleChange={visible => {
-                    this.setVisibilityState(visible);
-                }}
-                onPopupAlign={tooltipEl => setArrowLeft(tooltipEl, '22px')}
-            >
-                <button
-                    data-test="GlobalSettingsButton"
-                    style={{
-                        marginRight: 0,
-                        marginLeft: 5,
+        if (this.props.inFilterPanel) {
```

Comment:
what's difference between these two modes?  hard for me to see.

Reply:
`inFilterPanel` refers to if the button is in the filter panel. In the comparison view, the button is in the filter panel as opposed to the results view where the button is on the top of the page by the "Modify Query" button. If it is in the panel, the button's properties are set so it is more suitable for the filter panel, like setting its trigger to click, its tooltip placement is from the top, the icon is smaller, styling, etc.